LastModified isn't in ISO 8601

This is what I get from a GET call on a bucket
```xml
...
<LastModified>Tue, 13 Mar 2018 17:25:04 GMT</LastModified>
...
```

According to this stack trace using the last aws-java-sdk-s3 client
```
Caused by: java.lang.IllegalArgumentException: Invalid format: "Tue, 13 Mar 2018 15:54:25 GMT"
	at org.joda.time.format.DateTimeParserBucket.doParseMillis(DateTimeParserBucket.java:187)
	at org.joda.time.format.DateTimeFormatter.parseMillis(DateTimeFormatter.java:826)
	at com.amazonaws.util.DateUtils.doParseISO8601Date(DateUtils.java:98)
	at com.amazonaws.util.DateUtils.parseISO8601Date(DateUtils.java:77)
	at com.amazonaws.services.s3.internal.ServiceUtils.parseIso8601Date(ServiceUtils.java:76)
	at com.amazonaws.services.s3.model.transform.XmlResponsesSaxParser$ListBucketHandler.doEndElement(XmlResponsesSaxParser.java:703)
	at com.amazonaws.services.s3.model.transform.AbstractHandler.endElement(AbstractHandler.java:52)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:609)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1782)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2967)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:602)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:505)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:841)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:770)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
	at com.amazonaws.services.s3.model.transform.XmlResponsesSaxParser.parseXmlInputStream(XmlResponsesSaxParser.java:147)
```

and this
<img width="394" alt="capture d ecran 2018-03-13 a 18 28 37" src="https://user-images.githubusercontent.com/2699647/37358845-62565252-26ec-11e8-8135-33f25d6aa820.png">

I guessed that it should be ISO8601-stringified.

Do you agree?

--- a/lib/xml-template-builder.js
+++ b/lib/xml-template-builder.js
@@ -44,7 +44,7 @@ exports.buildBucketQuery = function(options, data) {
       ...data.objects.map(object => ({
         Contents: {
           Key: object.key,
-          LastModified: object.metadata["last-modified"],
+          LastModified: object.metadata["last-modified"].toISOString(),
           ETag: object.metadata["etag"],
           Size: object.size,
           StorageClass: "STANDARD",

--- a/lib/xml-template-builder.js
+++ b/lib/xml-template-builder.js
@@ -44,7 +44,7 @@ exports.buildBucketQuery = function(options, data) {
       ...data.objects.map(object => ({
         Contents: {
           Key: object.key,
-          LastModified: new Date(object.metadata["last-modified"]).toISOString(),
+          LastModified: object.lastModifiedDate.toISOString(),
           ETag: object.metadata["etag"],
           Size: object.size,
           StorageClass: "STANDARD",

